tests/Unit/Domain/Test_Domain.cpp

```diff
@@ -1130,8 +1128,6 @@ Domain<3> create_serialized_domain() {
 
   const std::vector<CoordinateMaps::Distribution> object_A_radial_distribution{
       CoordinateMaps::Distribution::Logarithmic};
-  const std::vector<CoordinateMaps::Distribution> object_B_radial_distribution =
-      object_A_radial_distribution;
```

Comment:
If you're going to keep this change, I'd rename `object_A_radial_distribution` to just `radial_distribution` so it's not confusing why below you have an `_A` thing used in a function call for `_B`.